Aptoide/aptoide-client-v8
# [ASV-677] [ASV-1211] Auto update to json refactor

**What does this PR do?**

 This PR is splitter in 3 parts:
- Refactor of the AutoUpdate flow (677)
- Change xml request to json requests (677)
- Adapt (add storeName for cobrand) to the Json request (1211)

**Database changed?**

   No

**Where should the reviewer start?**

AutoUpdateManager.java
MainActivity.java
MainPresenter.java

**How should this be manually tested?**

  Mock charles response to return an higher version code or make an OR in the autoUpdateManager if statement.
 Press Ok and check if the download starts 

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/ASV-677
https://aptoide.atlassian.net/browse/ASV-1211

**Code Review Checklist**
